config/config.dist.php

```diff
@@ -1,10 +1,13 @@
 <?php
+
+namespace Imbo\MetadataSearch;
+
 return [
     'eventListeners' => [
         'metadata' => [
             'listener' => 'Imbo\MetadataSearch\EventListener\MetadataOperations',
```

Comment:
When providing the listener class name as a string Imbo expects it to be a FQCN. We could provide it as an instance like we do with the backend, but I'm not sure if that's better or worse.